utils/find-overlay-dependencies.sh

```diff
@@ -64,9 +64,10 @@ CMAKE_DEPENDS_NAME[watchos]="SWIFT_MODULE_DEPENDS_WATCHOS"
 
 echo $1
 for sdk in ${(k)SDKS}; do
+  sdk=$sdk$SUFFIX
   arch=$SDKS[$sdk]
```

Comment:
Don't we still need the original name for looking up into SDKS and CMAKE_DEPENDS_NAME?